addons/godot_pixelorama_importer/pixelorama_file_open/openIn.gd

```diff
@@ -0,0 +1,74 @@
+extends EditorInspectorPlugin
+
+
+var pixel = "/home/luiq/Desktop/linux-64bit/"
```

Comment:
This probably needs changing to an editor settings variable :P

Reply:
Ah! That was my test variable . I will remove that later!